applications/Cyborg.md

```diff
@@ -0,0 +1,188 @@
+# Cyborg Network - Empowering Decentralized Edge Networks
+
+- **Team Name:** Cyborg Network
+- **Payment Address:** 15VNtavdKpRmSvGmjKevMaYvtifjNDrSSp4Bv9E2sVsVcnjg (DOT) 
```

Comment:
Typically we offer 30% vested DOT and the remaining 70% is paid in either USDT or USDC on the Asset Hub. Could  you change the first payment address to reflect one of these instead of DOT? Thank you!

Reply:
Thanks for the info @keeganquigley , I have updated the payment address